### ISSUE TITLE
TSError: src/baseConn.ts:533:9 - error TS2454: Variable 'inTypeId' is used before being assigned.

### ISSUE THREAD

**Issue by denizdogan** [edgedb/edgedb-js#635]
Running `yarn test` in `packages/driver` causes the error. It seems like an easy fix, but maybe I'm the only one running into it? I'm not sure why the code is even written in a way that allows this to happen, but at the same time the code is so old, maybe TS was different back then.

```
~/Code/edgedb/edgedb-js/packages/driver master*
❯ yarn test
yarn run v1.22.19
$ npx --node-options='--experimental-fetch' jest --detectOpenHandles
Error: Jest: Got error running globalSetup - /Users/deniz/Code/edgedb/edgedb-js/packages/driver/test/globalSetup.ts, reason: [TSError: src/baseConn.ts:533:9 - error TS2454: Variable 'inTypeId' is used before being assigned.

533     if (inTypeId == null || outTypeId == null) {
            ~~~~~~~~
src/baseConn.ts:533:29 - error TS2454: Variable 'outTypeId' is used before being assigned.

533     if (inTypeId == null || outTypeId == null) {
                                ~~~~~~~~~
src/baseConn.ts:619:9 - error TS2454: Variable 'cardinality' is used before being assigned.

619     if (cardinality == null || outCodec == null || inCodec == null) {
            ~~~~~~~~~~~]
    at runGlobalHook (/Users/deniz/Code/edgedb/edgedb-js/node_modules/@jest/core/build/runGlobalHook.js:117:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runJest (/Users/deniz/Code/edgedb/edgedb-js/node_modules/@jest/core/build/runJest.js:327:5)
    at async _run10000 (/Users/deniz/Code/edgedb/edgedb-js/node_modules/@jest/core/build/cli/index.js:344:7)
    at async runCLI (/Users/deniz/Code/edgedb/edgedb-js/node_modules/@jest/core/build/cli/index.js:199:3)
    at async Object.run (/Users/deniz/Code/edgedb/edgedb-js/node_modules/jest-cli/build/run.js:124:37)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by scotttrinh** [comment]
This is working for me locally, and obviously this runs in CI, so I'm not sure why you're encountering that issue. Are you sure you're on a clean checkout and at `master` with no changes?

**Comment by scotttrinh** [comment]
Going to close this until we see it pop up in CI or some other reproducible environment.

**Comment by scotttrinh** [comment]
This was fixed as a side-effect of #774

**Comment by denizdogan** [comment]
@scotttrinh Brilliant 🚀 😄
